anomalib/deploy/inferencers/openvino_inferencer.py

```diff
@@ -159,11 +159,16 @@ def post_process(
         if "image_threshold" in meta_data:
             pred_label = pred_score >= meta_data["image_threshold"]
 
-        if task == "segmentation":
+        if task == "classification":
+            _, pred_score = self._normalize(pred_scores=pred_score, meta_data=meta_data)
+        elif task == "segmentation":
             if "pixel_threshold" in meta_data:
                 pred_mask = (anomaly_map >= meta_data["pixel_threshold"]).astype(np.uint8)
 
-            anomaly_map, pred_score = self._normalize(anomaly_map, pred_score, meta_data)
+            anomaly_map, pred_score = self._normalize(
+                pred_scores=pred_score, anomaly_maps=anomaly_map, meta_data=meta_data
+            )
+            assert anomaly_map is not None
```

Comment:
Would it be an idea to add `else` here to raise a `ValueError` for the rest of the `task` values?